state_machine/applications/system-check/tests/logging_test.py

```diff
@@ -0,0 +1,85 @@
+"""
+Python system check script for PyCubed satellite board
+PyCubed Mini mainboard-v02 for Pocketqube Mission
+SD Card Logging Test
+* Author(s): Yashika Batra
+"""
+
+import os
+
+
+def sd_test():
+    """
+    Run a basic test; create a file and test existence, write to and
+    read from a file, delete file and test existence. Return result_dict
+    values accordingly
+    """
+
+    # create filepaths
+    fp = "test.txt"
+    test_fp = "/sd/" + fp
```

Comment:
Honestly I feel like using these variables makes things more confusing.
I think it would be easier to either use the literal string, or to change the variables to something that is a little more clear